vsc_pdf_template_editor/lib/src/application_service.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter_treeview/flutter_treeview.dart';
+
+abstract class IApplicationService{
+  Future<List<Node>> getListNode(List<Map<String, dynamic>> listNode);
+}
+class ApplicationService extends IApplicationService {
+
+  @override
+  Future<List<Node>> getListNode(List<Map<String, dynamic>> listNode) async {
```

Comment:
Again, in a later PR, I don't think this needs to be a Future. Also don't know what `ApplicationService` means.